beacon-chain/blockchain/service.go

```diff
@@ -44,10 +45,23 @@ func (c *ChainService) Start() {
 // Stop the blockchain service's main event loop and associated goroutines.
 func (c *ChainService) Stop() error {
 	defer c.cancel()
-	log.Info("Stopping blockchain service")
+	log.Info("Stopping service")
 	return nil
 }
 
+// ProcessBlock accepts a new block for inclusion in the chain.
+func (c *ChainService) ProcessBlock(b *types.Block) error {
```

Comment:
I think there's already a few block processing logic and state transition log implemented, we can try to merge those

Reply:
ah, true, there's already a channel i can feed the block into